seed.js

```diff
@@ -75,9 +88,65 @@ var seedProducts = function() {
             price: 0.25,
             brand: 'Applebees',
             description: 'Craft beer bespoke post-ironic gochujang direct trade shabby chic, tacos humblebrag fashion axe mlkshk twee typewriter chillwave meggings. Umami sriracha paleo post-ironic, blue bottle lumbersexual kickstarter mustache squid pork belly skateboard taxidermy. Echo park beard 90\'s tumblr you probably haven\'t heard of them 8-bit. Actually pabst 90\'s, chambray helvetica food truck kinfolk seitan typewriter raw denim. Crucifix irony pug, kogi seitan chia salvia intelligentsia letterpress green juice distillery. Godard tumblr shoreditch, kale chips swag pork belly fixie master cleanse aesthetic. Occupy neutra banh mi, four dollar toast roof party twee poutine biodiesel sartorial bushwick literally cronut keytar.',
-            quantity: 1
+            quantity: 5
+        },
+        {
+            name: 'Chicken Wing (Drummette)',
+            imageUrl: 'http://img1.21food.com/img/cj/2014/10/8/1412770556005517.jpg',
```

Comment:
since these are arrays now, they should be initialized as arrays i think. 

maybe have one of the products have 2 imageUrls